test/stat_tracker_test.rb

```diff
@@ -133,26 +133,32 @@ def test_it_can_get_rival
   end
 
   def test_it_can_get_winningest_coach
-    assert_equal "Dan Lacroix", @stat_tracker.winningest_coach("20122013")
+    assert_equal "Claude Julien", @stat_tracker.winningest_coach("20132014")
+    assert_equal "Alain Vigneault", @stat_tracker.winningest_coach("20142015")
   end
 
   def test_it_can_get_worst_coach
-    assert_equal "Martin Raymond", @stat_tracker.worst_coach("20122013")
+    assert_equal "Peter Laviolette", @stat_tracker.worst_coach("20132014")
+    assert_equal "Ted Nolan", @stat_tracker.worst_coach("20142015")
   end
 
   def test_it_can_get_most_accurate_team
-    assert_equal "DC United", @stat_tracker.most_accurate_team("20122013")
+    assert_equal "Real Salt Lake", @stat_tracker.most_accurate_team("20132014")
+    assert_equal "Toronto FC", @stat_tracker.most_accurate_team("20142015")
   end
 
   def test_it_can_get_least_accurate_team
-    assert_equal "New York City FC", @stat_tracker.least_accurate_team("20122013")
+    assert_equal "New York City FC", @stat_tracker.least_accurate_team("20132014")
+    assert_equal "Columbus Crew SC", @stat_tracker.least_accurate_team("20142015")
   end
 
   def test_it_can_get_team_with_most_tackles
-    assert_equal "FC Cincinnati", @stat_tracker.most_tackles("20122013")
+    assert_equal "FC Cincinnati", @stat_tracker.most_tackles("20132014")
+    assert_equal "Seattle Sounders FC", @stat_tracker.most_tackles("20142015")
```

Comment:
Good to see the test for stat_tracker using the full data. I changed some of the tests I added to do the same thing.